### PR TITLE
BZ-1682920: Pods-per-core references removed from OCP docs

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 3.11 only
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1682920
QE ack required.
Preview Links: 

- [Setting maximum pods per node](https://deploy-preview-43625--osdocs.netlify.app/openshift-enterprise/latest/admin_guide/manage_nodes.html#admin-guide-max-pods-per-node)
- [Node group definitions](https://deploy-preview-43625--osdocs.netlify.app/openshift-enterprise/latest/install/configuring_inventory_file.html#configuring-inventory-node-group-definitions)
- [Multiple Masters Using Native HA with External Clustered etcd](https://deploy-preview-43625--osdocs.netlify.app/openshift-enterprise/latest/install/example_inventories.html#multi-masters-using-native-ha-ai)

### CODE REVIEW

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -734,10 +734,10 @@ maximums] page for the maximum supported limits for each version of
 {product-title}.
 ====
 
-In the *_/etc/origin/node/node-config.yaml_* file, two parameters control the
-maximum number of pods that can be scheduled to a node: `pods-per-core` and
-`max-pods`. When both options are in use, the lower of the two limits the number
-of pods on a node. Exceeding these values can result in:
+In the *_/etc/origin/node/node-config.yaml_* file, one parameter controls the
```
Comment (mjpytlak) [inline admin_guide/manage_nodes.adoc:737]: I believe we treat file paths as a literal. Please use back ticks instead of bold.

Comment (sagidlow) [reply admin_guide/manage_nodes.adoc:737]: @mjpytlak this is for 3.11. OCP 3 docs had a different style than OCP 4 docs.

Comment (sagidlow) [reply admin_guide/manage_nodes.adoc:737]: @mjpytlak the original doc:https://docs.openshift.com/container-platform/3.11/admin_guide/manage_nodes.html#admin-guide-max-pods-per-node has the file path as bold and not a literal.

**Pull request merged**